duckduckgo/macos-browser
# June 17 embedded config/tracker files update

Task/Issue URL: https://app.asana.com/0/1199230911884351/1202467738548419/f
Tech Design URL:
CC:

**Description**:

This PR updates the embedded tracker and config files.



**Steps to test this PR**:
1. Run the app and check that tracker blocking works, and that the config is behaving as you would typically expect
1. Review the updates to the files and make sure they look okay. Reviewing the tracker data file is out of scope given its size, so be sure to do manual tracker blocking tests to ensure it's working.

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
